test/unit/lodash.spec.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+            describe('when first param is lower then second param', function ()
+            {
+                it('should return value equal second param', function ()
+                {
+                    var params = datasets.clamp1();
+                    expect(params[1]).to.be.below(params[2]);
```

Comment:
Your describe is saying first lower than second, but you are checking second and third.